qiskit_dynamics/models/generator_models.py

```diff
@@ -444,4 +450,4 @@ def _evaluate_in_frame_basis_with_cutoffs(self, sig_vals: Array):
         return 0.5 * (
             np.tensordot(sig_vals, self._ops_in_fb_w_cutoff, axes=1)
             + np.tensordot(sig_vals.conj(), self._ops_in_fb_w_conj_cutoff, axes=1)
-        )
+        )
```

Comment:
Could be slightly faster (by one element-wise multiplication) if we just define the operators to be half as large when passing them to the operator collections.